SpringProject/manipulability_test.m

```diff
@@ -42,6 +42,7 @@
 % draw robot to confirm (FK)
 
 
+state = zeros(1,36);
```

Comment:
switch indexing dims

Reply:
fixed in ce27c1abd3bb5b6b2af4460e46af67cfc407ce39